ARrehab/ARrehab/Minigames/ExpressionGame/Expressions.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 import UIKit
 import ARKit
 
+// What is the point of extending NSObject?
```

Comment:
Haven't tested it quite yet but lets try just removing it and seeing if it breaks? According to docs its for Objective-C? but not sure how swift ended up with it in the codebase still...